packages/loader/src/ktx2/transcoder/BinomialLLCWorkerCode.ts

```diff
@@ -91,7 +97,8 @@ export function transcode(buffer: Uint8Array, targetFormat: any, KTX2File: any):
     ETC,
     R8,
     RG8,
-    RGBA8
+    RGBA8,
+    BC6H
```

Comment:
Why not use `KTX2TargetFormat` directly?

Reply:
This function will convert to string for using in the webworker, so the outside `KTX2TargetFormat` cannot be imported by the worker.